app/cinema/hall.py

```diff
@@ -0,0 +1,18 @@
+from app.people.cinema_staff import Cleaner
+from app.people.customer import Customer
+
+
+class CinemaHall:
+    def __init__(self, number: int) -> None:
+        self.number = number
+
+    def movie_session(self,
+                      movie_name: str,
+                      customers: [Customer],
```

Comment:
It would be better to specify in the type annotation that you expect to obtain a list of `Customer` instances for `customers`
```suggestion
                      customers: list[Customer],
```